### ISSUE TITLE
Update documentation

### ISSUE THREAD

**Issue by rebeccabilbro** [DistrictDataLabs/yellowbrick#34]
specifically adding the roadmap and end goal description and depiction of the yellowbrick/scikit-learn workflow (maybe something like baleen has)

**Comment by bbengfort** [comment]
And a gallery!

**Comment by rebeccabilbro** [comment]
@bbengfort Working on addressing this very minimally right now, will push preliminary description to README momentarily.

**Comment by bbengfort** [comment]
@rebeccabilbro ok sounds good; I'm almost done with the examples notebook - have to do some minor massaging of the classifiers.

**Comment by rebeccabilbro** [comment]
@bbengfort Ok, pushed to README: https://github.com/DistrictDataLabs/yellowbrick/blob/develop/README.md. Here's the snippet:

## Using Yellowbrick
The Yellowbrick API is specifically designed to play nicely with Scikit-Learn. Here is an example of a typical workflow sequence with Scikit-Learn and Yellowbrick:

### Feature Visualization
In this example, we see how Rank2D performs pairwise comparisons of each feature in the data set with a specific metric or algorithm, then returns them ranked as a lower left triangle diagram.
```python
from yellowbrick.features import Rank2D

visualizer = Rank2D(features=features, algorithm='covariance')
visualizer.fit(X, y)                # Fit the data to the visualizer
visualizer.transform(X)             # Transform the data
visualizer.poof()                   # Draw/show/poof the data
```

### Model Visualization
In this example, we instantiate a Scikit-Learn classifier, and then we use Yellowbrick's ROCAUC class to visualize the tradeoff between the classifier's sensitivity and specificity.
```python
from sklearn.svm import LinearSVC
from yellowbrick import ROCAUC

model = LinearSVC()
model.fit(X,y)
y_pred = model.predict(X)
visualizer = ROCAUC(model)
visualizer.score(y,y_pred)
visualizer.poof()
```

For additional information on getting started with Yellowbrick, check out our [quick start guide](https://github.com/DistrictDataLabs/yellowbrick/blob/master/docs/setup.rst).

**Comment by bbengfort** [comment]
@rebeccabilbro looks great! The only thing I'd add is we probably want to point people to the [examples](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/examples/examples.ipynb) rather than the quick start guide which doesn't appear to be complete.
